html/blog.html

```diff
@@ -29,6 +32,86 @@
         </div>
     </header>
     <!--NAVIGATION BAR ENDS-->
+
+    <!--Main Content Begins-->
+    <div class="main-container">
+
+        <main>
+            <section class="Blog-Title">

```

Comment:
Keep class names lowercase
```suggestion
            <section class="blog-title">
```